mu-plugins/blocks/language-suggest/src/edit.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * The edit function describes the structure of your block in the context of the
+ * editor. This represents what the editor will render when the block is used.
+ *
+ * @return {WPElement} Element to render.
+ */
+export default function Edit() {
+	return (
+		<p { ...useBlockProps() }>
+			This is a suggestion, which usually <a href="http://wordpress.org">links</a> to something.
```

Comment:
This message might be confusing to someone skimming the page in the editor. How about…
```suggestion
			WordPress is also available in <a href="http://wordpress.org">language</a> (if applicable).
```